boards/egis/egis_et171/egis_et171_defconfig

```diff
@@ -0,0 +1,78 @@
+CONFIG_XIP=n
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+CONFIG_FLASH_BASE_ADDRESS=0x80000000
+
+# Serial driver options
+CONFIG_SERIAL=y
+
+# GPIO driver options
+CONFIG_GPIO=y
+
+# Syscon driver options
+CONFIG_SYSCON=y
+
+# hwinfo driver options
+CONFIG_HWINFO=y
+CONFIG_HWINFO_ANDES=y
+
+# RV64 options
+#CONFIG_RV64I_CPU=y
+
+# FPU (single precision) options
+#CONFIG_FPU=y
+#CONFIG_FPU_SHARING=y
+#CONFIG_FLOAT_HARD=y
+#CONFIG_SINGLE_PRECISION_FPU=y
+
+# FPU (double precision) options
+#CONFIG_FPU=y
+#CONFIG_FPU_SHARING=y
+#CONFIG_FLOAT_HARD=y
+#CONFIG_DOUBLE_PRECISION_FPU=y
+
+# SRAM size KB
+CONFIG_SRAM_SIZE=384
+
+# PMP options
+#CONFIG_PMP_GRANULARITY=8
+
+# SMP options
+#CONFIG_SMP=y
+#CONFIG_MP_MAX_NUM_CPUS=$(cores_num)
+
```

Comment:
why is any of this, or anything below here?

Reply:
@jackylee-go, I am updating adp_xc7k_ae350_defconfig in #85927, you may refer to it.